src/App.tsx

```diff
@@ -15,37 +16,91 @@
   'Garlic',
 ];
 
+enum Way {
+  'Unknown' = 0,
+  'Info' = 'is-info',
+  'Success' = 'is-success',
+}
+
+function isValidWay(value: Way): value is Way {
+  return Object.values(Way).includes(value);
+}
+
+function getSorted(obj: string[], way: Way) {
+  const currentGoods = [...obj];
+
+  currentGoods.sort((el1, el2) => {
+    switch (way) {
+      case Way.Info:
+        return el1.localeCompare(el2);
+      case Way.Success:
+        return el1.length - el2.length;
+      default:
+        return 0;
+    }
+  });
+
+  return currentGoods;
+}
+
 export const App: React.FC = () => {
+  const [selectItem, setSelectItem] = useState('');
+  const [selReverse, setselReverse] = useState(false);
```

Comment:
The state function `setselReverse` should follow camelCase naming conventions. Consider renaming it to `setSelReverse` for consistency.